src/v2/Apps/Artwork/Utils/badges.ts

```diff
@@ -0,0 +1,11 @@
+export const shouldRenderAuthenticityCertificate = artwork => {
+  return artwork.hasCertificateOfAuthenticity && !artwork.is_biddable
+}
+
+export const shouldRenderVerifiedSeller = artwork => {
+  return !artwork.is_biddable && artwork.partner?.isVerifiedSeller
+}
+
+export const shouldRenderGuarantee = artwork => {
+  return artwork.is_acquireable || artwork.is_offerable
+}
```

Comment:
**question**: Do we lost any type safety by moving this functions into a separate utility file? Is there a way to explicitly type the artwork argument if it can't be inferred? I imagine this properties rely on the GraphQL query quite a bit.

Reply:
We can do something like this

```javascript
interface Artwork {
  hasCertificateOfAuthenticity?: boolean | null
  is_biddable?: boolean | null
  is_acquireable?: boolean | null
  is_offerable?: boolean | null
  partner?: {
    isVerifiedSeller?: boolean | null
  } | null
}

export const shouldRenderAuthenticityCertificate = (artwork: Artwork) => { ... }

export const shouldRenderVerifiedSeller = (artwork: Artwork) => { ... }

export const shouldRenderGuarantee = (artwork: Artwork) => { ... }
```

Reply:
Do you mean remove `null` and `undefined` from `Artwork` interface? 

```javascript
interface Artwork {
  hasCertificateOfAuthenticity: boolean
  is_biddable: boolean
  is_acquireable: boolean
  is_offerable: boolean
  partner: {
    isVerifiedSeller: boolean
  }
}
```

If yes, then this will complicate our task, since not all data will be available in all badge components (for example, for `SecurePayment` only `is_acquireable` and `is_offerable` fields are [queried](https://github.com/artsy/force/blob/1a73bf4bfeb9df6517f1178735aa7079aab6f7a5/src/v2/Apps/Artwork/Components/TrustSignals/SecurePayment.tsx#L48-L49) and we will have to manually specify all the others

Reply:
aah, got it. My bad, sorry 😅

Reply:
Nothing to be sorry for! Thanks for clarifying 👍🏼